docs/overview.rst

```diff
@@ -54,9 +55,9 @@ Use cases
 
 .. note::
 
-    At Mozilla, *Kinto* is used in *Firefox* and *Firefox OS* for global synchronization
-    of settings and assets, as well as a first-class solution for personal data in
-    browser extensions and Web apps.
+    At Mozilla, *Kinto* is used in *Firefox* for global synchronization
+    of frequently changed settings and the Web Extensions storage.sync API;
+    as well as in *Firefox for Android* for ab-testing and downloading of extra assets.
```

Comment:
Also, "downloading of extra assets" is kind of confusing to me (what makes an asset extra? why are we downloading it anyhow?). I think Kinto's strongest use case to date is blocklists, so maybe reword to that instead?